.gitignore

```diff
@@ -25,3 +25,5 @@ build/Release
 # Dependency directory
 # https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
 node_modules
+
+config/
```

Comment:
I don't think we should be ignoring an entire folder. Also, is a single `config.js` in the root not enough? Perhaps, do you think there's so many options we should have multiple files?